addons/trenches/CfgWorlds.hpp

```diff
@@ -0,0 +1,383 @@
+    class SaraLite {
+        surfaceTextureBasePath = "ca\saralite\Data";
+        filePrefix = "detail_co.paa";
+        camouflageObjects[] = {"A3\plants_f\Clutter\c_Thistle_Thorn_Brown.p3d"};
+    };
+
+    //Staszow
+    class Staszow  {
+        surfaceTextureBasePath = "WW2\TerrainsIF_t\Worlds\IF_Surfaces_t\staszow";
+        filePrefix = "co.paa";
+        camouflageObjects[] = {};
+    };
+
+    //Stratis
+    class Stratis {
+      surfaceTextureBasePath = "a3\map_data";
+      filePrefix = "co.paa";
+      camouflageObjects[] = {"A3\plants_f\Clutter\c_Thistle_Thorn_Brown.p3d"};
+    };
+
+    //Takistan
+    class Takistan {
+        surfaceTextureBasePath = "ca\takistan\Data";
+        filePrefix = "co.paa";
+        camouflageObjects[] = {};
+    };
+
+    //Takistan Mountains
+    class Mountains_ACR {
+        surfaceTextureBasePath = "ca\afghan\Data";
+        filePrefix = "co.paa";
+        camouflageObjects[] = {};
+    };
+
+    //Tanoa
+    class Tanoa {
+        surfaceTextureBasePath = "A3\Map_Data_Exp";
+        filePrefix = "co.paa";
+        camouflageObjects[] = {"A3\Vegetation_F_Exp\Clutter\Grass\c_GrassBunch_HI.p3d"};
+    };
+
+    //United Sahrani
+    class Sara_dbe1  {
+      surfaceTextureBasePath = "ca\sara\Data";
+      filePrefix = "detail_co.paa";
+      camouflageObjects[] = {"A3\plants_f\Clutter\c_Thistle_Thorn_Brown.p3d"};
+      class Surfaces {
+          class mesto {
+              texturePath = "ca\sara\Data\mesto_co.paa";
+          };
+      };
+    };
+
+    //Utes
+    class utes {
+        surfaceTextureBasePath = "ca\utes\Data";
+        filePrefix = ".paa";
+        camouflageObjects[] = {"CUP\Terrains\cup_terrains_worlds\Clutter\c_GrassCrookedForest.p3d"};
+        class Surfaces {
+            class Default {
+                texturePath = "a3\map_data\gdt_beach_co.paa";
+            };
+        };
+    };
+
+    //Zargabad
+    class Zargabad {
+        surfaceTextureBasePath = "ca\zargabad\Data";
+        filePrefix = "co.paa";
+        camouflageObjects[] = {"ca\plants_E\Clutter\c_Brush_Soft_EP1.p3d"};
+    };
+
+    //Ihantala Winter
+    class tem_ihantalaw {
+        surfaceTextureBasePath = "tem\tem_ihantalaw\data";
+        filePrefix = "co.paa";
+        camouflageObjects[] = {};
+    };
+
+    //Rosche
+    class WL_Rosche {
+        surfaceTextureBasePath = "WL\WL_Rosche\data";
+        filePrefix = "_co.paa";
+        camouflageObjects[] = {};
+    };
+
+    //Abramia
+    class abramia {
+        surfaceTextureBasePath = "ibr\abramia\data";
+        filePrefix = "detail_co.paa";
+        camouflageObjects[] = {};
+    };
+
+    //Isla Duala
+    class isladuala3 {
+        surfaceTextureBasePath = "ibr\isladuala3\data";
+        filePrefix = "detail_co.paa";
+        camouflageObjects[] = {};
+    };
+
+    //Tembelan
+    class tembelan {
+        surfaceTextureBasePath = "ZI\Tembelan\data";
+        filePrefix = "_co.paa";
+        camouflageObjects[] = {};
+    };
+};
```

Comment:
This would mean that there a many maps now in the game when using ACE. Think of a different way to store this information. CBA namespace maybe or just some sort of associative array.

Reply:
Whats the problem with having it this way?

Reply:
Too vulnerable to 3rd party edits. Breaks scripts that read this class and expect the maps to actually exist. Adds untagged tokens to public classes. It violates my personal quality standards, so it will have to be changed before this PR can be merged.
If you're unable to do it yourself, have patience with me. I may get to this over christmas.

Reply:
If I would name the config file differnetly whould that be change enough or does it have to be cba namespace?

Reply:
cba namespace is something entirely different than config. 
Yeah changing names would be enough. prefix with ace_trenches